EpiAware/src/EpiObsModels/StackObservationModels.jl

```diff
@@ -77,7 +80,7 @@ Generate observations from a stack of observation models. Assumes a 1 to 1 mappi
 
     obs = ()
     for (model, model_name) in zip(obs_model.models, obs_model.model_names)
-        @submodel prefix=eval(model_name) obs_tmp=generate_observations(
+        @submodel obs_tmp = generate_observations(
             model, y_t[Symbol(model_name)], Y_t[Symbol(model_name)])
         obs = obs..., obs_tmp...
```

Comment:
This seems an unusual way to `reduce` over a collection? You're recursively generating an immutable `Tuple`. `mapreduce` is smart enough to produce a tuple give tuple input, I'm not sure why you've gone a different way here?

Reply:
I don't think I know what these words mean? Can we port to an issue?